packages/menu/src/Menu.ts

```diff
@@ -63,7 +63,10 @@ export class Menu extends SpectrumElement {
     }
 
     public focus(): void {
-        if (!this.menuItems.length) {
+        if (
+            !this.menuItems.length ||
+            !this.menuItems.filter((item) => !item.disabled).length
```

Comment:
You maybe could use menuItems.every(..) here to make it more obvious what you are testing.

Reply:
Good idea. Is `!this.menuItem.some((item) => !item.disabled)` similarly clear, or no? It has the possibility of not having to traverse the entire array.

Reply:
Well. `every` returns a boolean and you don´t have to test for length again. And `every` would break immediately if the condition is not met. So you don´t iterate over the whole array if you don´t need to.
So `this.menuItem.every((item) => item.disabled)` would test if all items are disabled.